test/models/report_test.rb

```diff
@@ -3,7 +3,27 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @report = FactoryBot.create(:report)
+    @user = FactoryBot.create(:user)
+  end
+
+  test 'user should edit own report' do
+    user = @report.user
+    assert @report.editable?(user)
```

Comment:
report を作成した user であることが分かるように以下のように書いてもよいかなと思います。`user` 変数だと `@user` 変数との区別がちょっと難しいという問題もあります。

```suggestion
    assert @report.editable?(@report.user)
```

Reply:
ありがとうございます！
おっしゃる通り`@report.user`の方がreportを作成した本人が編集をできるというテストにも合致していますね。